### PR TITLE
SonicTT16B Update #1

### PR DESCRIPTION
**Pull request by JeodC**
This update adds support for the 32bit apk of this game. The launchscript also unpacks and repacks the apk for optimization, and adds a texture compression patch for 64-bit.

### CODE REVIEW

**Comment by JeodC** [comment]
Do not merge yet

**Review by kloptops — APPROVED** [review]

**Comment by kloptops** [comment]
lol, checked over the changes before i read the comment about not merging. 😁

**Comment by JeodC** [comment]
Ok to review and merge now.

**Review by kloptops — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**
